nix/shells.nix

```diff
@@ -21,7 +21,7 @@ let
       buildInputs = with pkgs; [
         clojure flock maven openjdk
         # lint specific utilities
-        clj-kondo zprint clojure-lsp
+        clj-kondo zprint clojure-lsp ripgrep
```

Comment:
Hi @jakubgs, is it ok to put this here? Or into the default shell.

Reply:
Yes, that's correct.